### ISSUE TITLE
Events logged in events.log and openhab.log

### ISSUE THREAD

**Issue by lolodomo** [openhab/openhab-distro#224]
n OH2, ItemStateChangedEvent are logged twice, in events.log and in openhab.log.
In OH1.x, they were only logged in events.log.
Is it the expected behaviour ?

Is there a setting to have events only logged in events.log ?

**Comment by kaikreuzer** [comment]
Yes, I agree it should not be in openhab.log, but it should be on the console when doing a "log:tail".
I thought I had configured it to work this way by https://github.com/openhab/openhab-distro/blob/master/distributions/distribution-resources/src/main/resources/etc/org.ops4j.pax.logging.cfg#L6, but then it seems it does not work as expected.
If you find out, what to change in this file, please create a PR - thanks

**Comment by joergkling** [comment]
I did some digging and it looks like it works as designed. Openhab.log is the default log and therefore logs everything. Event.log is an additional file which logs only specific items. So in order to remove the duplicate entries the ones in openhab.log would need to be filtered. I tried to get filtering to work but struggle a little.

This is what I have done so far:

`log4j.appender.out.filter.f1=smarthome
log4j.appender.out.filter.f1.LevelMin=FATALE
log4j.appender.out.filter.f1.LevelMax=TRACE
log4j.appender.out.filter.f1.AcceptOnMatch=false`

Will continue to investigate. However this is all new to me and might take some time. So any help would be highly appreciated.
